static/app/views/performance/transactionSummary/transactionOverview/content.tsx

```diff
@@ -498,4 +526,15 @@ const StyledSearchBar = styled(SearchBar)`
   }
 `;
 
+const StyledWarningIcon = styled(IconWarning)`
+  color: ${p => p.theme.alert.warning.iconColor};
+  height: ${p => p.theme.iconSizes.sm};
+  width: ${p => p.theme.iconSizes.sm};
```

Comment:
I think you can set the size and color prop on the icon and you shouldn't need to style it

Reply:
ah nice, kk added 👍